src/app/kvm/components/KVMDetailsHeader/KVMDetailsHeader.test.tsx

```diff
@@ -1,93 +1,63 @@
-import { mount } from "enzyme";
-import { Provider } from "react-redux";
-import { MemoryRouter } from "react-router-dom";
-import configureStore from "redux-mock-store";
+import { screen } from "@testing-library/react";
 
 import KVMDetailsHeader from "./KVMDetailsHeader";
 
+import urls from "app/base/urls";
 import { KVMHeaderViews } from "app/kvm/constants";
-import type { RootState } from "app/store/root/types";
-import {
-  pod as podFactory,
-  podResources as podResourcesFactory,
-  podState as podStateFactory,
-  podStatus as podStatusFactory,
-  podStatuses as podStatusesFactory,
-  podVmCount as podVmCountFactory,
-  rootState as rootStateFactory,
-} from "testing/factories";
-
-const mockStore = configureStore();
+import { getTestState, renderWithBrowserRouter } from "testing/utils";
 
 describe("KVMDetailsHeader", () => {
-  let state: RootState;
+  let state: ReturnType<typeof getTestState>;
```

Comment:
Is the return type of getTestState not RootState?

(testing/utils.tsx line 220)

Reply:
That's exactly it. Using the `ReturnType` just removes an additional line of import.